cordapp-example/contracts-kotlin/src/main/kotlin/com/example/contract/CashContract.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.contract
+
+import com.example.state.CashState
+import net.corda.core.contracts.CommandData
+import net.corda.core.contracts.Contract
+import net.corda.core.contracts.requireSingleCommand
+import net.corda.core.contracts.requireThat
+import net.corda.core.transactions.LedgerTransaction
+
+class CashContract : Contract {
+    companion object {
+        @JvmStatic
+        val ID = CashContract::class.java.canonicalName
+    }
+
+    override fun verify(tx: LedgerTransaction) {
+        val command = tx.commands.requireSingleCommand<Commands>()
+
+        when (command.value) {
+            is Commands.Create -> requireThat {
+                // everyone can print/crate cash... because of covi... crisis
```

Comment:
Instead of determining the name of who is allowed to create cash, how about you record who created the cash, and let people decide what is real money and what is monopoly money?